packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -187,9 +189,9 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
             const overridden = this.overriddenPreferenceName(preferenceName);
             if (overridden) {
                 delete this.overridePatternProperties.properties[`[${overridden.overrideIdentifier}]`];
-                delete this.combinedSchema.properties[`[${overridden.overrideIdentifier}]`];
+                this.removePropFromSchemas(`[${overridden.overrideIdentifier}]`);
             } else {
-                delete this.combinedSchema.properties[preferenceName];
+                this.removePropFromSchemas(preferenceName);
```

Comment:
schema is also manipulated in `updateOverridePatternPropertiesKey`. should manipulate the two new fields there as well?

Reply:
`patternProperties` is only used in generateTree() from combinedSchema (which generates the Preference page tree) so I thought it is redundant to update it also for workspaceSchema/folderSchema. 
If you think I should update it anyway - let me know and I'll do it.
![image](https://user-images.githubusercontent.com/71069170/102008391-a3c52000-3d38-11eb-8d2e-f6047238d003.png)